quickwit-proto/src/cluster.rs

```diff
@@ -4,36 +4,39 @@
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct Member {
     //// Member ID.　A string of the UUID.
-    #[prost(string, tag = "1")]
+    #[prost(string, tag="1")]
```

Comment:
seems that something changed in the proto auto generated files. Is this on purpose?

Reply:
This is a side effect of updating tonic / prost.